### PR TITLE
Add Hastebin command

### LINKED ISSUE #476: Command that sends code to Hastebin
https://github.com/ImGabe/ElyxBot/blob/master/comandos/hastebin.js

### PR DESCRIPTION
**Pull request by pedrofracassi**
+ Closes #476

**Needs to be fixed before merging:**
- [x] Parameters ignore line breaks

### CODE REVIEW

File: src/commands/utility/hastebin.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+module.exports = class Hastebin extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'hastebin'
+    this.aliases = ['haste']
+
+    this.category = 'utility'
+
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ full: true, missingError: 'commands:npm.noNameProvided' })
```
Comment (dpaiv0) [inline src/commands/utility/hastebin.js]: npm?

File: src/commands/utility/hastebin.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const { CommandStructures, SwitchbladeEmbed, Constants } = require('../../')
```
Comment (dpaiv0) [inline src/commands/utility/hastebin.js]: unused var "Constants"

**Comment by davipatury** [comment]
To be merged after #606

**Comment by metehus** [comment]
ready ehwn

**Comment by Doges** [comment]
RFR

**Pull request merged**